keyboards/tmd/alicia/readme.md

```diff
@@ -0,0 +1,17 @@
+# TMD Alicia
+
+![Alicia](https://i.imgur.com/ctOAq1l.jpg)
+
+A TGR-Alice-compatible homage PCB (with SMT per-key RGB and/or RGB strips) - designed for use with themillerdave's TMD Alicia stacked aluminum cases
+
+* Keyboard Maintainer: [themillerdave](https://github.com/themillerdave)  
+* Hardware Supported: ATMEGA32U4 custom designed PCB   
+* Hardware Availability: hit up /u/themillerdave on reddit or email dave@themmillerdave.com
+
+Make example for this keyboard (after setting up your build environment):
+
+    make tmd/alicia:default  # (for Windows usage)
+    # OR
+    make tmd/alicia:mac # (for those of us who prefer Apple computers)
```

Comment:
I'm not sure if this helps,  but bootmagic has Alt-Gui and Contrl-Gui swap, which may be useful and not need another keymap

Reply:
I use the bootmagic function on many of my other keyboards - which is awesome for taking my keyboard to work where I use Windows (at home I prefer Macs), allowing me to swap Alt and Control without reflashing the board.  However, on this particular layout, the options are somewhat limited for the bottom row (and while it may be personal preference:) the differences between these keymaps is more than a simple swap of Alt and control.